test/models/report_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.